### ISSUE TITLE
Frequently error on generate songs

### ISSUE THREAD

**Issue by iamfourh2e** [gochendong/suno-api#20]
I always got error whenever i generated 5 requests or some time fews . i need to go to suno.com to recreate a song. so i can generate it again. is this a hCapcha problem or else ?

**Comment by Legus-Yeung** [comment]
yes, I have the same issue as well. It's also noted in the readme that you have to do so
